### ISSUE TITLE
Able to change a hostname in item detail view

### ISSUE THREAD

**Issue by eliserichards** [mozilla-lockwise/lockwise-android#1165]
### Needs reproduce on actual device!
I'm seeing this on my emulator

## Steps to reproduce
1. Open a login to the detail view
2. Click to copy username or password
3. Start typing

### Expected behavior
Not able to edit the login

### Actual behavior
Able to edit the login: 
<img width="417" alt="image" src="https://user-images.githubusercontent.com/43795363/73651544-ef249180-4684-11ea-98e7-27c47ca58503.png">

Note that this is happening on my emulator when the device's keyboard is closed - I'm guessing this won't be reproducible on an actual device.

### Device & build information
* Device: Pixel 2 emulator
* Build version: v4.0.0 Build 6013

**Comment by caugner** [comment]
@eliserichards I don't think this really is an issue for *actual* devices, because you usually cannot type on actual devices without a keyboard (which only opens when focussing an input field first).

The only exception could be bluetooth/USB keyboards.
